geth/api/api.go

```diff
@@ -195,6 +195,7 @@ func (api *StatusAPI) JailBaseJS(js string) {
 	api.b.jailManager.BaseJS(js)
 }
 
+// Notify (push notification and mailbox??)
```

Comment:
Notify sends a push notifications to the device with the given token.

Reply:
Got it.